deploy/helm-chart/templates/config.yaml

```
@@ -4,17 +4,17 @@ metadata:
   name: {{ include "promscale.fullname" . }}
   namespace: {{ template "promscale.namespace" . }}
   labels:
-    app: {{ template "promscale.fullname" . }}
+    app: {{ include "promscale.fullname" . }}
     chart: {{ template "promscale.chart" . }}
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
-    app.kubernetes.io/name: "promscale-connector"
+    app.kubernetes.io/name: "promscale"
```

```suggestion
    app.kubernetes.io/name: { include "promscale.fullname" . }}
```

Just an idea that came to my mind. What about having this particular label be the same as `app` label?

---

I thought about this yesterday as well.  I will make the change.

---

I just refactored how the labels are added to each template.  It should make it easier to manage in the future.